### ISSUE TITLE
Integration of sqlcl into yum.oracle.com repo

### ISSUE THREAD

**Issue by Nico-DB** [oracle/oracle-db-tools#33]
Dear Oracle,
In our company we use the oracle repo yum.oracle.com to install oracle instant client versions for sqlplus and tools. We would like to use sqlcl too. It is possible that you upload or integrate the sqlcl tools in oracle instant client or somewhere else that we can download sqlcl from yum.oracle.com too? Currently it is a manuell process to download the sources.

**Comment by krisrice** [comment]
I did try to get it there once and was told no. Let me try again and see if
they'll let it in this time. FWIW, there is a sqlcl rpm in the oci yum
repository so on cloud compute nodes one can simply "yum install sqlcl" (
and ords is there too )

-kris

**Comment by krisrice** [comment]
So it comes down to the license currently on sqlcl requiring a click-thru. Once we change that ( in progress ) we can put it there.

**Comment by bamcgill** [comment]
Should have closed this a while ago, but we are all set now on yum.oracle.com 
@Nico-DB take a look now.
